### DISCUSSION TITLE
Help with a TMUX binding

### DISCUSSION THREAD

**Discussion by al3xandru** [pqrs-org/Karabiner-Elements#3696] [Q&A]
TMUX uses `Ctrl-b` as a prefix for its keybindings.  I am using Goku to define some complex modifications.  The one I am looking for suggestions is for the TMUX `Ctrl-b x` which kills a panel but asks for `y` confirmation.

I have tried 

```
:simlayers {
    :b-tmux-mode {:key :b}  ;; this enables keeping the `b` key down and pressing another key
}

{:des "Tmux"
  :rules [
      [:x [:!Tb :x :y]]
   ]}
```

The attempt was hoping to send `Ctrl-b x` then followed by the `y` needed for the confirmation.  This doesn't seem to work as the `y` is somehow sent to the remaining panel in tmux.

Please share any suggestions/ideas/feedback

**Comment by amlanjlahkar** [comment]
Very early response but by default the `kill-pane` function is prefixed with `confirm-before` for both <prefix>& and <prefix>x. You can just rebind x to `bind x kill-pane` to bypass the confirmation. No need setting up an external keymap for that
